lib/src/flowcontrol/stream_queues.dart

```diff
@@ -166,10 +166,10 @@ class StreamMessageQueueIn extends Object
   final Queue<Message> _pendingMessages = Queue<Message>();
 
   /// The [StreamController] used for producing the [messages] stream.
-  StreamController<StreamMessage> _incomingMessagesC;
+  late StreamController<StreamMessage> _incomingMessagesC;
```

Comment:
Again, StreamControllers can usually be created eagerly, and then just have `onListen` attached later, which can avoid a `late`.
(Could the variable be final?).